open_academy/models/open_academy_course.py

```diff
@@ -7,4 +6,7 @@ class OpenAcademyCourse(models.Model):
     _description = "open.academy.course.description"
 
     name = fields.Char(string="Course", required=True)
-    description = fields.Text(string="Course description", required=True)
+    description = fields.Text(required=True)
+
+    responsible_id = fields.Many2one('res.users')
+    session_id = fields.One2many('open.academy.session', 'course_id')
```

Comment:
```suggestion
    session_ids = fields.One2many('open.academy.session', 'course_id')
```
See [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#38fields) for fields